dev-libs/boehm-gc/boehm-gc-8.2.0.ebuild

```diff
@@ -9,7 +9,8 @@ MY_P="gc-${PV}"
 
 DESCRIPTION="The Boehm-Demers-Weiser conservative garbage collector"
 HOMEPAGE="https://www.hboehm.info/gc/ https://github.com/ivmai/bdwgc/"
-SRC_URI="https://github.com/ivmai/bdwgc/releases/download/v${PV}/${MY_P}.tar.gz"
+SRC_URI="https://github.com/ivmai/bdwgc/releases/download/v${PV}/${MY_P}.tar.gz
```

Comment:
While I appreciate this only affects loong, still needs a revbump or you'll have broken installs with a missing header,

Reply:
It's true in the generic sense, but there is very likely no other existing loong installs with this version installed... everyone knowing to upgrade to this keyword-less version is presumably already on #gentoo-loong. This way a rebuild for nearly everyone else can be saved...

Reply:
There is no anybody else as it's not keyworded.